cooperator/models/partner.py

```diff
@@ -105,6 +105,10 @@ def _compute_share_info(self):
         store=True,
         compute="_compute_representative_of_member_company",
     )
+    type = fields.Selection(
+        selection_add=[("representative", "Representative")],
+        ondelete={"representative": "set null"},
+    )
```

Comment:
I am missing a comment here that explains _why_ we need this type in `cooperator`.

Reply:
comment added

Reply:
The comment is not very precise, because it is not very clear if this field is indeed needed (see the discussion in the issue). The problem is also explained in the roadmap.

Reply:
also added in 12.0

Reply:
@victor-champonnois I would do one of two things (or both):

1. Copy the explanation from the discussion verbatim into the comment.
2. Provide a link to the discussion in the comment.

Having this context would be really handy when/if we hit this problem again.

Reply:
:thumbsup: